deployment/index.ts

```diff
@@ -163,6 +163,8 @@ const usage = deployUsage({
   docker,
   environment,
   tokens,
+  redis,
```

Comment:
I wonder about adding these - is there a way we can avoid it? 

Redis should be fine, since it's in-cluster for now and it does not have issues, even if something external fails. 
But, Postgres means that now usage service depends on a new external thing, and I wonder if we should avoid that? (in the past, we did try to avoid introducing more complexity/deps to usage service) 

@n1ru4l @kamilkisiela thoughts?

Reply:
The usage service already independently depends upon postgres, but via the tokens service. I could implement this via the tokens service again and add more complexity. We decided to eventually kill the tokens service as it is unnecessary.

Reply:
as long as we deduplicate `getOrSet` bento calls and have `redis` to share tokens between `usage` replicas, we're good